Source/core/inspector/JavaScriptCallFrame.h

```diff
@@ -42,7 +42,16 @@
 namespace blink {
 
 class JavaScriptCallFrame : public RefCountedWillBeGarbageCollectedFinalized<JavaScriptCallFrame>, public ScriptWrappable {
+#ifdef DISABLE_INSPECTOR
```

Comment:
write something like this in Source/bindings/core/v8/ScriptWrappable.h:
ifndef DISABLE_INSPECTOR
define DEFINE_WRAPPERTYPEINFO ---> previous definition.
...
#else
# Move your code here.
#endif